aws/templates/id/id_lambda.ftl

```diff
@@ -90,16 +90,20 @@
 [#function getFunctionState occurrence]
     [#local core = occurrence.Core]
 
+    [#assign id = formatResourceId(LAMBDA_FUNCTION_RESOURCE_TYPE, core.Id) ]
+
     [#return
         {
             "Resources" : {
                 "function" : {
-                    "Id" : formatResourceId(LAMBDA_FUNCTION_RESOURCE_TYPE, core.Id),
+                    "Id" : id,
                     "Name" : formatSegmentFullName(core.Name)
                 }
             },
             "Attributes" : {
-                "REGION" : regionId
+                "REGION" : regionId,
+                "ARN" : getExistingReference("id",ARN_ATTRIBUTE_TYPE)!"",
```

Comment:
No need for default - getExistingReference returns "" if the id isn't found. Also id shouldn't be in quotes